src/components/member-profile/index.js

```diff
@@ -351,12 +359,22 @@ const Profile = (props) => {
   };
 
   return (
-    <>
+    <div
+      role="button"
+      tabIndex={0}
+      onKeyDown={(e) => {
+        if (e.ctrlKey || e.metaKey) {
+          setIsOptionKey(true);
+        }
```

Comment:
In [HomePage](https://github.dev/vinayak-trivedi/website-members/blob/a489d33d2264767a74c8e5841f1f25bb46c33c18/src/components/home/index.js#L29) we are checking for the `Alt` key to call `setIsOptionKey(true)` and here we're using `ctrlKey` or `metaKey`. 
Should we keep it same at both places or it was intended to be different?
If we can keep it same then we should also extract this to a util method

Reply:
It was mentioned in the issue so I did that, but you are right it would be better if we have the same key

Reply:
I checked it in the home component there is a lot of prop drilling that is happening and in my case I am taking it from usercontext, I will change that button to Alt but the refactor can be done in a future PR.

Reply:
Hey samarpan I tried using the Alt key but it is creating a weird behaviour in chrome, when I press the alt key it shifts the focus to customize and control google chrome features, so for the first time it works but if the user tries to access the buttons again with pressing that alt key the setting icon doesn't show up. 
Image for reference⬇️
![image](https://user-images.githubusercontent.com/90315175/185786526-559497b8-d656-4d00-81bc-2532e9f8d71f.png)